include/llama/macros.hpp

```diff
@@ -52,7 +52,8 @@
 #endif
 
 #ifndef LLAMA_UNROLL
-#    if defined(__NVCC__) || defined(__NVCOMPILER) || defined(__clang__) || defined(__INTEL_LLVM_COMPILER)
+#    if defined(__HIP__) || defined(__NVCC__) || defined(__NVCOMPILER) || defined(__clang__)                          \
+        || defined(__INTEL_LLVM_COMPILER)
```

Comment:
Since `__HIP__` is also handled via `__clang__` here, either remove the `__HIP__` here or add it to the definitions of other macros in this file (e.g. `LLAMA_FORCE_INLINE`).

Reply:
I added it to `LLAMA_FORCE_INLINE` too

Reply:
Great, thx!